### PR TITLE
feat: show command

### PR DESCRIPTION
**Pull request by eugene-davis**
Adds a show command that lets you see the number of failures of a test over a specified duration, i.e. `show <testname> over 1d`

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: ebr_trackerbot/command/show.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        config["api_url"],
+        params={
+            "test_status": "failed",
+            "start": start.to_iso8601_string(),
+            "end": pendulum.now("UTC").to_iso8601_string(),
+        },
+        headers={"accept": "application/json"},
+    )
+
+    channel_id = payload["data"]["channel"]
```
Comment (martinm82) [inline ebr_trackerbot/command/show.py:33]: Is it guaranteed that these items are always in the payload?

File: ebr_trackerbot/command/show.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        )
+        return
+
+    for test in response.json()["tests"]:
+        full_name = test["full_name"]
+        if full_name == target_test:
+            count = str(test["count"])
+            payload["web_client"].chat_postMessage(
+                channel=channel_id,
+                text="Test *{test}* failed {count} times over {duration}".format(
```
Comment (martinm82) [inline ebr_trackerbot/command/show.py]: Shouldn't there be a time unit after `{duration}`?

And I guess it should be something like `... over last {duration} seconds`?

Comment (eugene-davis) [reply ebr_trackerbot/command/show.py]: It is providing the duration string back that the user provided, so they have provided the units.

File: ebr_trackerbot/command/show.py (hunk 1 above)
Comment (eugene-davis) [reply ebr_trackerbot/command/show.py:33]: Yes, this comes from slack.

File: ebr_trackerbot/utility.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+"""
```
Comment (ghost) [inline ebr_trackerbot/utility.py]: I would name this file more specific like time_utility.py

Comment (eugene-davis) [reply ebr_trackerbot/utility.py]: Done

**Review by ghost — APPROVED** [review]

**Pull request merged**